includes/PSpellSpellChecker.php

```diff
@@ -48,8 +48,9 @@ public function getSuggestions($lang, $words) {
 
 		$outWords = array();
 		foreach ($words as $word) {
-			if (!pspell_check($plink, trim($word))) {
-				$outWords[] = utf8_encode($word);
+            $word = trim($word);
+            if (!pspell_check($plink, $word)) {
+                $outWords[utf8_encode($word)] = pspell_suggest($plink, $word);

```

Comment:
this will give broken output if pspell returns something else then utf-8 (which is the case in many languages)